packages/@react-spectrum/slider/docs/RangeSlider.mdx

```diff
@@ -0,0 +1,152 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/slider';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/slider/package.json';
+
+```jsx import
+import {Flex} from '@react-spectrum/layout';
+import {RangeSlider} from '@react-spectrum/slider';
+```
+
+---
+category: Forms
+---
+
+# Slider - RangeSlider
+
+<p>{docs.exports.RangeSlider.description}</p>
+
+<HeaderInfo
+packageData={packageData}
+componentNames={['RangeSlider']}
+sourceData={[{type: 'Spectrum', url: 'https://spectrum.adobe.com/page/slider/'}]} />
+
+## Example
+
+```tsx example
+<RangeSlider label="Range" defaultValue={{start: 12, end: 36}} />
+```
+
+## Value
+
+RangeSliders are controlled with the `value` and `defaultValue` props.
+This value consists of two parts, start and end, both must fall between
+the RangeSlider's minimum and maximum values, which default to 0 and 100 respectively.
```

Comment:
Feels like this would flow better as two sentences.
```suggestion
This value consists of two parts, `start` and `end`. Both parts must fall between
the RangeSlider's minimum and maximum values, which default to 0 and 100 respectively.
```
or if we wanna keep it as a single sentence
```suggestion
This value consists of two parts, start and end, both of which must fall between
the RangeSlider's minimum and maximum values, which default to 0 and 100 respectively.
```